modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx

```diff
@@ -163,7 +163,7 @@ export function RightSidebarObjectDefinitionDetails({
 			catch (error: unknown) {
 				const {message} = error as Error;
 
-				openToast({message, type: 'danger'});
+				openToast({autoClose: 15000, message, type: 'danger'});
```

Comment:
Please `import {openToast} from 'frontend-js-web'` and use it directly.

Reply:
just did it for this and the other files, thank you!!